cmd/note_common.go

```diff
@@ -166,42 +165,64 @@ func createCommitComments(project string, mrID int, commit string, body string,
 			}
 		} else if strings.HasPrefix(scanner.Text(), "|") {
 			if comments != "" {
-				createCommitNote(project, mrID, commit, newfile, oldfile, oldLineNum, newLineNum, comments, block)
+				createCommitNote(project, mrID, commit, newfile, oldfile, linetype, oldLineNum, newLineNum, comments, block)
 				comments = ""
 			}
 			// read line numbers
-			fs := strings.Split(scanner.Text(), " ")
+			f := strings.Split(strings.Join(strings.Fields(strings.TrimSpace(scanner.Text())), " "), " ")
+			if f[1] == "@@" {
+				continue;
```

Comment:
gofmt should remove the ;
I see that we are still using travis, which is basically dead AFAIK. We should either move to something like GH actions or move to gitlab straight away

Reply:
Damned c coding style.  It's my fingers :) I can't get away from it.  Fixed.